src/components/structures/auth/Registration.tsx

```diff
@@ -269,7 +269,7 @@ export default class Registration extends React.Component<IProps, IState> {
         );
     }
 
-    private onUIAuthFinished = async (success, response, extra) => {
+    private onUIAuthFinished = async (success, response) => {
```

Comment:
types?

Reply:
I followed the trail for `response` and it's set as `any` in js-sdk